README.md

```diff
@@ -86,15 +86,19 @@ selected identities can read the data.
 To store the encrypted string `Hello` which only I can read into an account:
 ```
 ECPublicKey myPublicKey = api.getMyPublicKey();
-Data data = new DataBuilder().bytes("Hello".getBytes()).addReader(myPublicKey)
-			.build();
+Data data = new DataBuilder()
+    .bytes("Hello".getBytes())
```

Comment:
Really should include a character set here, aka `getBytes(StandardCharsets.UTF_8)` or whatever is appropriate.

Reply:
Cool, will fix on master branch.